resources/js/Pages/Authors.vue

```diff
@@ -0,0 +1,165 @@
+<script setup>
+import Papa from 'papaparse'
+import moment from 'moment'
+import { computed, ref, watch } from 'vue'
+import { useLogsStore } from '@/Stores/logsStore'
+import { Head } from '@inertiajs/vue3'
+
+const logsStore = useLogsStore()
+const logs = computed(() => logsStore.getLogs)
+
+const colors = [
+  'bg-gray-400',
+  'bg-blue-400',
+  'bg-red-400',
+  'bg-green-400',
+  'bg-yellow-400',
+  'bg-indigo-400',
+  'bg-purple-400',
+  'bg-pink-400',
+] 
```

Comment:
This const is repeating in many files, you could simply add your colors to tailwind.config.js 


look at this: https://tailwindcss.com/docs/configuration